mkdocs.yml

```
@@ -61,9 +61,12 @@ nav:
   - Guides:
     - User Guides:
       - Getting started: guides/index.md
+      - Use Cases:
+        - Serve Multiple GenAI models: guides/serve-multiple-genai-models
+        - Serve Multiple LoRA adapters: guides/serve-multiple-lora-adapters
```

Hopefully this works - current nav is not per https://deploy-preview-812--gateway-api-inference-extension.netlify.app/guides/
```suggestion
        - Serve Multiple GenAI models: guides/serve-multiple-genai-models.md
        - Serve Multiple LoRA adapters: guides/serve-multiple-lora-adapters.md
```